<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QLCDNumber" name="lcdTimeCount">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="smileButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdMineCount">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuStart">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="init"/>
    <addaction name="choice"/>
    <addaction name="rank"/>
   </widget>
   <widget class="QMenu" name="menuhelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionabout"/>
    <addaction name="actiondescription"/>
   </widget>
   <addaction name="menuStart"/>
   <addaction name="menuhelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="init">
   <property name="text">
    <string>初始化</string>
   </property>
  </action>
  <action name="rank">
   <property name="text">
    <string>排行榜</string>
   </property>
  </action>
  <action name="choice">
   <property name="text">
    <string>选项</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actiondescription">
   <property name="text">
    <string>游戏说明</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
